### DISCUSSION TITLE
Is it possible to add a reorderable() method ?

### DISCUSSION THREAD

**Discussion by loukasstoltz** [awcodes/filament-curator#501] [Ideas]
![Capture d’écran 2024-06-28 à 14 17 33](https://github.com/awcodes/filament-curator/assets/19878463/3146a2ab-3021-4a1f-8181-364fdfaec2ba)

Thank you kindly :-)

**Comment by awcodes** [comment]
Reorderable isn't built into the plugin because it doesn't really make sense the context of a media library where there could be 1000s of files.

If you truly need it you could extend the Model and ListMedia Resource and add an order column to the db table to add that functionality yourself.

Also, it looks like you haven't added the plugin styles to a custom theme either. That is covered in the first part of the read me. :)

**Comment by loukasstoltz** [comment]
I have do that ... 
What's wrong ? 
I run the 'vite' command line, nothing change ...
![Capture d’écran 2024-06-28 à 15 29 20](https://github.com/awcodes/filament-curator/assets/19878463/ab5126fe-7743-46ec-94c1-1cd6b2acf08b)
![Capture d’écran 2024-06-28 à 15 29 27](https://github.com/awcodes/filament-curator/assets/19878463/8afe3f07-fb62-4dc5-9d6e-015e7ef2c165)

**Reply by xalabama** [reply]
Did you have added the viteTheme inside your service provider panel config?

**Comment by loukasstoltz** [comment]
Thank you kindly ! 
It work better now :-) 

![Capture d’écran 2024-07-01 à 09 36 51](https://github.com/awcodes/filament-curator/assets/19878463/c3954423-aed1-43c8-b029-11385610d365)
